packages/game-client/src/App.jsx

```diff
@@ -168,6 +175,16 @@ function App() {
 		directions.current[snakeId] = direction;
 	};
 
+	//gets all the hashes of snakes and returns it as an object :)
+
+	const getSnakeHash = (state) => {
+		const combinedHash = Object.values(state).reduce((result, obj) => {
+		Object.assign(result, obj.hash);
+		return result;
+		}, {});
```

Comment:
Your git hooks are not installed properly... That is why lint-staged in not running prettifying your commits.